srcpkgs/mopidy/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'mopidy'
 pkgname=mopidy
-version=3.0.1
+version=3.2.0
 revision=4
```

Comment:
```suggestion
revision=1
```

Reply:
I added this change to the new revision :+1: